src/harness/reference_models/ppa_gwpz_fss_gwbl_purge/ppa_gwpz_fss_gwbl_purge.py

```diff
@@ -0,0 +1,86 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+"""
+===================================================================================
+  The main function is ppaGwpzFssPlusGwblPurgeReferenceModel(). The grants from FAD
+  CBSD objects of SAS UUT and SAS test harnesses are purged if the CBSD lies within
+  a list of PPA areas or GWPZ areas or within 150 KMs from a list of FSSs and the
+  grant frequency range overlaps with the protected entity's frequency range.
+===================================================================================
+"""
+from shapely.geometry import shape, Point, Polygon
+from reference_models.geo import vincenty
+from reference_models.pre_iap_filtering import pre_iap_util
+
+FSS_GWBL_PROTECTION_FREQ_RANGE = {'lowFrequency': 3650000000,
+                             'highFrequency': 3700000000}
+
+
+def ppaGwpzFssPlusGwblPurgeReferenceModel(sas_uut_fad, sas_test_harness_fads,\
```

Comment:
Can you change the function name which is not really readable, into something easier to read ?
Also can we remove the \ at the end which is useless?

Reply:
updated   the function name to 'zonePurgeReferenceModel'